.github/workflows/upload-sbom.yaml

```diff
@@ -28,13 +26,13 @@ jobs:
         uses: actions/download-artifact@v4-beta
         with:
           run-id: ${{ github.event.workflow_run.id }}
-          name: edgebit-agent.spdx.json
+          name: sbom.spdx.json
 
       - name: Upload SBOM to EdgeBit
         uses: edgebitio/edgebit-build@v1
         with:
           edgebit-url: https://edgebit.edgebit.io
           token: ${{ secrets.EDGEBIT_TOKEN }}
           component: edgebitio-edgebit-agent
-          sbom-file: ${{ steps.sbom.outputs.download-path }}
-          args-file: ${{ steps.metadata.outputs.download-path }}
+          sbom-file: ${{ steps.sbom.outputs.download-path }}/metadata.json
+          args-file: ${{ steps.metadata.outputs.download-path }}/sbom.spdx.json
```

Comment:
Are these two swapped?

Reply:
To elaborate a little bit: my read of this is that the suffix on `sbom-file` should be `/sbom.spdx.json` and the suffix on `args-file` should be `/metadata.json`. The stuff inside the `${{ ... }}` on both lines looks correct.

Reply:
yep, great catch!